src/sharepoint/order/sharepoint-order.controller.js

```diff
@@ -22,33 +29,48 @@ angular
         main: 'sharepoint.alerts.main',
       };
 
+      this.isReseller = false;
+      this.associateExchange = false;
       this.associatedExchange = null;
+      this.accountsToAssociate = [];
+
       // Indicates that we are activating a SharePoint by coming from an exchange service.
-      this.isCommingFromAssociatedExchange = false;
+      this.isComingFromAssociatedExchange = false;
+
       this.loaders = {
         init: true,
+        accounts: true,
       };
-      this.worldPart = this.constants.target;
-      this.associateExchange = true;
+
       this.standAloneQuantity = 1;
-      this.accountsToActivate = [];
 
       this.getExchanges()
+        .then(() => this.User.getUser())
+        .then((user) => { this.userSubsidiary = user.ovhSubsidiary; })
+        .then(() => this.checkReseller())
+        .then((isReseller) => {
+          this.isReseller = isReseller;
+
+          // default mode for normal users is to associate
+          if (!isReseller || this.isComingFromAssociatedExchange) {
+            this.associateExchange = true;
+          }
+        })
         .finally(() => {
           this.loaders.init = false;
-          this.associatedExchange = _.first(this.exchanges);
+          this.associatedExchange = this.associatedExchange || _.first(this.exchanges);
```

Comment:
```suggestion
           this.associatedExchange = this.associatedExchange || _.head(this.exchanges);
```
https://lodash.com/docs/4.17.11#head